native/ripper/ripper_test.go

```diff
@@ -300,76 +300,76 @@ func TestRipperLexFail(t *testing.T) {
-	}
-	for i, tt := range tests {
-		evaluated := vm.ExecAndReturn(t, tt.input)
-		vm.VerifyExpected(t, i, evaluated, tt.expected)
-	}
-}
+//func TestRipperInstruction(t *testing.T) {
```

Comment:
@hachi8833 can you take a look at this when you got a chance? It's too verbose to fix so I have to comment them out for now

Reply:
Got it

Reply:
Looks the recent commits might affect this. I will check this again when your refactoring finishes.

Reply:
Ok I finished changes, you can check that when you're free